bitbots_navigation/bitbots_path_planning/bitbots_path_planning/planner.py

```diff
@@ -94,3 +100,31 @@ def get_path(self) -> Path | None:
         Returns the most recent path
         """
         return self.path
+
+
+class DummyPlanner(Planner):
+    def __init__(self, node: Node, buffer: tf2.Buffer, map: Map) -> None:
+        super().__init__(node, buffer, map)
+
+    def step(self) -> Path:
+        return self.get_path()
+
+    def get_path(self) -> Path:
+        pose = PoseStamped()
+        my_position = self.get_my_position()
+        pose.pose.position.x = my_position.x
+        pose.pose.position.y = my_position.y
+
+        self.path = Path(
+            header=Header(frame_id=self.map.get_frame(), stamp=self.node.get_clock().now().to_msg()),
+            poses=[pose, self.goal],
+        )
+
+        return self.path
+
+
+def planner_factory(node: Node) -> type:
```

Comment:
```suggestion
def planner_factory(node: Node) -> Planner:
```
I think, the return type should be Planner

Reply:
No we return a type, that is instantiated by the user :D